### PR TITLE
Last set of changes for easier code readability, cleanliness and fixes

### PR DESCRIPTION
**Pull request by alejandroEsc**
Additional removal of panics.
Simplified how kraken commands are run, should make it fairly easy and standard now.
Changes to simplify more code.

**What this PR does / why we need it**:
Additional removal of panics.
Simplified how kraken commands are run, should make it fairly easy and standard now.
Changes to simplify more code.

This is preliminary work for me to write out a few more Unit Tests. It also simplifies helper code so that when we can potentially leverage future work against this code base.

**Release note**:

``` NONE
```

### CODE REVIEW

**Comment by alejandroEsc** [comment]
Adding a bug fix.

**Comment by alejandroEsc** [comment]
@coffeepac @davidewatson @yenicapotediaz  ready for review

**Comment by alejandroEsc** [comment]
resolves https://github.com/samsung-cnct/k2cli/issues/95

**Comment by alejandroEsc** [comment]
@davidewatson currently not much testing exists. However, some does exists and, except for some currently failing gke things, not sure why?, the current tests pass. I would add unit tests here but it would make checking this a bit more tedius. Rather start in a clean sleight now, rebase existing PRs and then add tests.

**Comment by davidewatson** [comment]
Fair enough. Since it has never passed CI, can you rebase to kick off another CI run and ensure you have all possible fixes. The last run failed when all of the k8s nodes failed to come up after etcd came up successfully.

**Comment by alejandroEsc** [comment]
@davidewatson checks passed.

**Comment by davidewatson** [comment]
Lgtm

File: cmd/cluster_down.go [hunk 1]
```diff
@@ -33,70 +31,34 @@ var downCmd = &cobra.Command{
...
+		spinnerPrefix := fmt.Sprintf("Bringing down cluster '%s' ", getFirstClusterName())
 
 		command := []string{
 			"ansible-playbook",
 			"-i",
 			"ansible/inventory/localhost",
 			"ansible/down.yaml",
 			"--extra-vars",
-			"config_path=" + ClusterConfigPath + " config_base=" + outputLocation + " config_forced=" + strconv.FormatBool(configForced) + " kraken_action=down ",
+			fmt.Sprintf("config_path=%s config_base=%s config_forced=%t kraken_action=down", ClusterConfigPath, outputLocation, configForced),
```
Comment (davidewatson) [inline cmd/cluster_down.go]: Nice.

**Pull request merged**